learningresources/tests/test_api.py

```diff
@@ -55,6 +57,11 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        self.assertRaises(
+            IntegrityError,
+            lambda: create_course(None, self.repo.id, 4, 9, self.user.id)
+        )
```

Comment:
Probably because a course with the pk of 4 doesn't exist in the database. I think it's not clear because the test is using magic numbers. It should be more explicit. Also, despite the name, `course_number` is an alphanumeric field.